WooCommerce/WooCommerceTests/Extensions/IconsTests.swift

```diff
@@ -168,6 +168,10 @@ final class IconsTests: XCTestCase {
         XCTAssertNotNil(UIImage.loginSiteAddressInfoImage)
     }
 
+    func testLoginJetpackErrorIconIsNotNil() {
```

Comment:
Nit: rename the test following our [new convention](https://github.com/woocommerce/woocommerce-ios/blob/develop/docs/naming-conventions.md#test-methods)